doc/examples/run_returning_interaction.rb

```diff
@@ -0,0 +1,53 @@
+# Example 3: RunReturningInteraction
```

Comment:
Same here - we don't need an order to the examples, and we shouldn't have any Niiwin code or references to Niiwin at all. Think of them as unrelated - this is a gem we developed independently and Niiwin just happens to be the first place we are using it.